lib/components/viewers/route-viewer.js

```diff
@@ -113,6 +109,53 @@ class RouteViewer extends Component {
   }
 }
 
+const StyledRouteRow = styled.div`
+  background-color: ${props => props.isActive ? '#f6f8fa' : 'white'};
+  border-bottom: 1px solid gray;
+`
+
+const RouteRowButton = styled(Button)`
+  padding: 8px;
+  width: 100%;
+`
+
+const RouteRowElement = styled.div`
+  display: inline-block;
+`
+
+const OperatorImg = styled.img`
+  height: 25px;
+  margin-right: 8px;
```

Comment:
The operator image appears to be offset vertically slightly from the other row elements:

![image](https://user-images.githubusercontent.com/2370911/99301909-a5282900-281c-11eb-80cd-305269ae7e4b.png)

Reply:
Fixed in 576730fe82ce0ae5ffac6e30c6f011b71b93a888